docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -353,12 +353,16 @@ Use the `freightMetadata()` template function to retrieve issue keys stored by p
 
 ```yaml
 # Access the default issue key
-issueKey: ${{ freightMetadata(ctx.targetFreight.name, 'jira-issue-key') }}
+issueKey: ${{ freightMetadata(ctx.targetFreight.name).jira-issue-key }}
```

Comment:
Dashes in attributes are something to be careful of. expr allows it, but it feels extremely awkward because most languages wouldn't permit this.

I would suggest favoring this alternative syntax here:

```suggestion
issueKey: ${{ freightMetadata(ctx.targetFreight.name)['jira-issue-key'] }}
```